app/scripts/lib/auth-errors.js

```diff
@@ -145,6 +145,48 @@ define(function (require, exports, module) {
       errno: 135,
       message: t('Unable to deliver email')
     },
+    // Secondary Email errors
+    EMAIL_EXISTS: {
+      errno: 136,
+      message: t('This email was already verified by another user')
+    },
+    EMAIL_DELETE_PRIMARY: {
+      errno: 137,
+      message: t('Can not delete primary email')
+    },
+    SESSION_UNVERIFIED: {
+      errno: 138,
+      message: t('Unable to perform action, unverified session')
+    },
+    EMAIL_PRIMARY_EXISTS: {
+      errno: 139,
+      message: t('Secondary email must be different than your account email')
+    },
+    EMAIL_VERIFIED_PRIMARY_EXISTS: {
+      errno: 140,
+      message: t('Account already exists')
+    },
+    UNVERIFIED_PRIMARY_EMAIL_NEWLY_CREATED: {
+      errno: 141,
+      message: t('Account already exists')
+    },
+    LOGIN_WITH_SECONDARY_EMAIL: {
+      errno: 142,
+      message: t('Primary account email required for sign-in')
+    },
+    SECONDARY_EMAIL_UNKNOWN: {
+      errno: 143,
+      message: t('Cannot use this email to sign in')
```

Comment:
we need to check this one, is this user facing?  from mtg: it should probably say `'Primary account email required for sign-in'`